### DISCUSSION TITLE
Merge queue: `github.base_ref` is not available in the action

### DISCUSSION THREAD

**Discussion by HHK1** [community/community#40277] [Pull Requests]
### Select Topic Area

Bug

### Body

I've been trying the merge queue beta feature.

I've added the `merge_group` option to my action triggers containing my required status checks.
When this is run, the PR is removed from the queue because it failed, and this is because the `github.base_ref` is returning an empty value.

How can I access an equivalent of the base_ref? I'm using lerna in a monorepo, and I'm using the base ref to know what operations need to be run.

**Comment by mrmckeb** [comment]
We hit this issue today, and it caused a lot of chaos.

For now, we're using `${{ github.base_ref || github.event.merge_group.base_ref }}`.

For more, see:
https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#merge_group

**Reply by HHK1** [reply]
Unfortunately it's not strictly equivalent. After testing the `merge_group.base_ref` is equal to the full ref i.e. `refs/heads/main`.
When using `pull_request.base.ref` or `gitHub.base_ref` the result is just the branch name `main`.

**Reply by mrmckeb** [reply]
Good point, and I agree that this should be resolved.

**Reply by oabutbul1** [reply]
Any workaround? thanks!

**Comment by HHK1** [comment]
Any updates on this? Is this taken into account as a bug?

**Reply by jlin880** [reply]
I'm still seeing this issue now, is there any ETA on this?

**Comment by radovanZRasa** [comment]
We are also hitting a roadblock because of this.
Intermediary branches which are created by GitHub for merge queue are not accessible from CI.

**Comment by payaaam** [comment]
Hi All! We are also experiencing this issue. Did anyone come up with a solution here?

**Comment by tomerkay** [comment]
`BASE_BRANCH=$(basename "${{ github.event.merge_group.base_ref }}")` as a workaround did the job for me

**Reply by k-capehart** [reply]
> `BASE_BRANCH=$(basename "${{ github.event.merge_group.base_ref }}")` as a workaround did the job for me

Thanks for this @tomerkay
For whatever reason `github.event.merge_group.base_ref` showed as invalid syntax in my editor, so I wasn't trying it. But I can confirm this worked for me!
